controllers/gcm/gcm.py

```diff
@@ -104,12 +104,11 @@ def __init__(self):
         self.LOCALHOST = False
         self.SERVER_KEY = Sitevar.get_by_id('gcm.serverKey')
         if self.SERVER_KEY is None:
-            raise Exception("Missing sitevar: gcm.serverKey. Can't send GCM messages.")
+            raise Exception("Missing sitevar: gcm.serverKey. Can't send FCM messages.")
         self.GCM_CONFIG = {'gcm_api_key': self.SERVER_KEY.contents['gcm_key']}
         self.GOOGLE_LOGIN_URL = 'https://www.google.com/accounts/ClientLogin'
         # Can't use https on localhost due to Google cert bug
-        self.GOOGLE_GCM_SEND_URL = 'https://android.apis.google.com/gcm/send'
-        self.GOOGLE_GCM_SEND_URL = 'https://android.googleapis.com/gcm/send'
+        self.GOOGLE_GCM_SEND_URL = 'https://fcm.googleapis.com/fcm'
```

Comment:
Whoops, this should be `https://fcm.googleapis.com/fcm/send`

Fixed in 3c72a6d1d90510192b3bfa68d8678691f776ebb7